droid-hal-device.inc

```diff
@@ -544,12 +544,21 @@ mkdir -p $RPM_BUILD_ROOT/boot
 mkdir -p $RPM_BUILD_ROOT/lib/modules
 
 # Install tools
+if [ $android_version_major -ge "11" ]; then
+if [ -e %{android_root}/system/tools/mkbootimg/mkbootimg.py ]; then
+    _mkbootimg=mkbootimg.py
+fi
+install -m 755 \
+    -D %{android_root}/system/tools/mkbootimg/${_mkbootimg:-mkbootimg} \
+    %{buildroot}/%{_bindir}/mkbootimg
+else
```

Comment:
To avoid code repetition and unindented multi-level `if` blocks, I'd suggest to just have a short
```bash
 if [ $android_version_major -ge "11" ]; then
    mkbootimg_dir=tools
else
    mkbootimg_dir=core
fi
```

Reply:
Done